desktop-environment.el

```diff
@@ -41,6 +41,60 @@
   :group 'environment)
 
 
+;;; Customization - keyboard backlight
+
+(defcustom desktop-environment-keyboard-set-command (concat "dbus-send "
```

Comment:
can you please use "keyboard-backlit" everywhere instead of just "keyboard"? This makes function and variable names very long but also much more explicit (which I prefer).

Reply:
Done!